app/models/practice.rb

```diff
@@ -52,39 +50,19 @@ def self.cached_published_enabled_approved_practices
   end
 
   def clear_searchable_cache
-    cache_keys = ["searchable_practices_json", "searchable_public_practices_json", "s3_signer", "published_enabled_approved_practices"]
-    cache_keys.each do |cache_key|
-      Cache.new.delete_cache_key(cache_key)
+    cached_keys = [
+      "searchable_practices_json",
+      "searchable_public_practices_json",
+      "s3_signer",
+      "published_enabled_approved_practices",
+      "#{cache_key}/as_json"
+    ]
+
+    cached_keys.each do |cached_key|
+      Rails.cache.delete(cached_key)
     end
   end
 
-  def clear_searchable_cache_on_save
-    if self.name_changed? ||
-        self.tagline_changed? ||
-        self.description_changed? ||
-        self.summary_changed? ||
-        self.initiating_facility_changed? ||
-        self.main_display_image_updated_at_changed? ||
-        self.published_changed? ||
-        self.enabled_changed? ||
-        self.approved_changed? ||
-        self.date_initiated_changed? ||
-        self.maturity_level_changed? ||
-        self.overview_problem_changed? ||
-        self.overview_solution_changed? ||
-        self.overview_results_changed?  ||
-        self.retired_changed? ||
-        self.retired_reason_changed? ||
-        self.hidden_changed? ||
-        self.is_public_changed?
-      self.reset_searchable_cache = true
-    end
-  end
```

Comment:
Let's dig in to the git blame and history for this a bit before removing:
https://github.com/department-of-veterans-affairs/diffusion-marketplace/blame/master/app/models/practice.rb

Reply:
@camillevilla I'm not really seeing any indication as to why Aurora went with this method, but I think that since all the values checked represent attributes on the the table it's a safe bet that utilizing `after_commit` will achieve the same end result. 

The only difference will be that it will now clear the cache if any attribute is updated, rather than just this selection (`practices` has a lot of columns). But I think that the slight performance lost from having an Innovation's cache cleared a bit more often will be negligible and that the reduction in complexity is worth it, but that is up for debate of course.